src/common/News/NewsList.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import {
+  FlatList,
+  View,
+  ActivityIndicator,
+} from 'react-native';
+import { useNavigationParam } from 'react-navigation-hooks';
+import NewsListItem from './NewsListItem';
+import commonStyles from '../styles';
+import { fetchStockNews } from './newsService';
+import ErrorMessage from '../ErrorMessage';
+import Card from '../Card';
+
+const NewsList = () => {
+  const query = useNavigationParam('query');
+  const [news, setNews] = useState([]);
+  const [pageNumber, setPageNumber] = useState(2);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isLoadingMore, setIsLoadingMore] = useState(false);
+  const [error, setError] = useState();
+  const [laodingMoreError, setLoadingMoreError] = useState();
+
+  const fetchInitList = () => {
+    setIsLoading(true);
+    setError(undefined);
+    fetchStockNews(query, 1)
```

Comment:
I'm not using redux because this component might be rendered simultaneously on both investmentsStack and watchlistsStack, I've already created a hook for this